docs/src/api/class-elementhandle.md

```diff
@@ -59,6 +59,7 @@ ElementHandle instances can be used as an argument in [`method: Page.$eval`] and
 ## async method: ElementHandle.$
 * langs:
   - alias-python: query_selector
+  - alias-csharp: QuerySelectorAsync
```

Comment:
I think a better solution would be to consistently rename it to `querySelector` in all of our mds, then alias it into `$` only in JS

Reply:
Going through that now, I *really* wish we'd use a language that would let me say "Find All References" :-)